stacks-common/src/util/pipe.rs

```diff
@@ -364,7 +353,7 @@ mod test {
             let expected_recv_buf = send_bytes.clone();
 
             pipe_read.set_nonblocking(true);
-            pipe_write.write(&send_bytes[..]).unwrap();
```

Comment:
The use of `write` in this test module is deliberate.

Reply:
ah kk will update :) seems strange in a test to only want to write up to some number of bytes and not ensure that number of bytes and panic otherwise.